src/js/components/Comments.js

```diff
@@ -74,7 +74,7 @@ class Comments extends Component {
   }
 
   renderUserComments(userToCommentsMap) {
-    return userToCommentsMap[this.props.currentUserId].map((comment) => {
+    return userToCommentsMap[this.props.selectedUserId].map((comment) => {
```

Comment:
Oof

Reply:
if it makes you feel better, you originally had selectedUserId, but because of the order that state updated in, selectedUserId wasn't set, so Alex changed it to currentUserId. 

I just put it back like you had it 😊